resources/js/records.js

```diff
@@ -0,0 +1,27 @@
+import { MAX_FILE_SIZE_BYTES } from './utils.js';
+
+/**
+ * Validates the file size when a file is selected
+ *
+ * @param {HTMLInputElement} input
+ */
+function validateFileSize(input) {
+    const file = input.files[0];
+
+    if (file && file.size > MAX_FILE_SIZE_BYTES) {
```

Comment:
Looks like this needs to be a `>=` instead of `>`. When I use a file that's exactly 100 MiB, it isn't caught by the client side logic and causes the server error.

```suggestion
    if (file && file.size >= MAX_FILE_SIZE_BYTES) {
```